File: shoppingm8_fe/lib/list/invitation/dto/ListInvitationRequestDto.dart
class ListInvitationRequestDto {
  final num invitedId;

  ListInvitationRequestDto(this.invitedId);

  Map<String, dynamic> toJson() {
    Map<String, dynamic> map = Map();
    map['invitedId'] = invitedId;
    return map;
  }
}

File: ShoppingM8-be/src/main/java/edu/pw/shoppingm8/list/invitation/api/dto/ListInvitationRequestDto.java
package edu.pw.shoppingm8.list.invitation.api.dto;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;
import lombok.Value;

@Value
@NoArgsConstructor(force = true, access = AccessLevel.PRIVATE)
@AllArgsConstructor
public class ListInvitationRequestDto {
    Long invitedId;
}
